eng/Versions.props

```
@@ -45,7 +45,7 @@
     <!-- Roslyn -->
     <MicrosoftCodeAnalysisVersion>3.3.1</MicrosoftCodeAnalysisVersion>
     <MicrosoftCodeAnalysisForRoslynDiagnosticsAnalyzersVersion>3.7.0</MicrosoftCodeAnalysisForRoslynDiagnosticsAnalyzersVersion>
-    <MicrosoftCodeAnalysisVersionForTests>4.0.1</MicrosoftCodeAnalysisVersionForTests>
+    <MicrosoftCodeAnalysisVersionForTests>4.1.0-3.22075.3</MicrosoftCodeAnalysisVersionForTests>
```

4.1.0 is now released.
```suggestion
    <MicrosoftCodeAnalysisVersionForTests>4.1.0</MicrosoftCodeAnalysisVersionForTests>
```